src/components/ContentSidebar/SidebarSection.js

```diff
@@ -12,7 +12,7 @@ import './SidebarSection.scss';
 type Props = {
     children?: any,
     className: string,
-    title: string | React$Element<any>,
+    title?: string | React$Element<any>,
```

Comment:
Made this optional so we can put details info inside a section with out the title/bar.